influxql/parser.go

```diff
@@ -577,6 +577,15 @@ func (p *Parser) parseSelectStatement(tr targetRequirement) (*SelectStatement, e
 		return nil, err
 	}
 
+	// Set if the query is a raw data query or one with an aggregate
+	stmt.IsRawQuery = true
```

Comment:
I think I might feel strongly about this. For example, look at this?

https://github.com/influxdb/influxdb/blob/master/influxql/ast.go#L756

Why is this then a method and not a property? My analogy may be poor, and if so I don't understand the code please let me know.